tasks/sapnote/2526952.yml

```diff
@@ -5,6 +5,11 @@
 
 # Configure tuned to use profile "sap-netweaver"
 
+- name: Show note number and version
+  debug:
+    msg: "SAP note {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '2526952') | first).number }}
+          (version {{ (__sap_netweaver_preconfigure_sapnotes_versions | selectattr('number', 'match', '2526952') | first).version }}): RHEL for SAP packages"
```

Comment:
You must use `selectattr('number', 'match', '^2526952$')` in order to do an exact match for the string `'2526952'` otherwise you'll match things like `xxx2526952` or `2526952xxx`